### ISSUE TITLE
Several functions are not exported from index.js

### ISSUE THREAD

**Issue by amaan-ahmad** [Susmita-Dey/all-round-calculator#26]
### Name

Amaan Ahmad

### Describe your Issue

Do Factorial is not exported from index.js

here is a snippet from `index.js`  
[reference](https://github.com/Susmita-Dey/all-round-calculator/blob/3ffe4585a897704d80b1379f3569e6a9daa3d677/package/index.js#L2) 

```
const doFactorial = require("./Functions/Factorial");
```

but like all other functions, this function is not exported

You can verify this by trying out the following code, it will result in an error.
```
const calculator = require('all-round-calculator');
console.log(calculator.dofactorial(5));
```

**Comment by Susmita-Dey** [comment]
@amaan-ahmad Do you want to work on this issue?

**Comment by amaan-ahmad** [comment]
Sure.

**Comment by Susmita-Dey** [comment]
> Sure.

Go ahead 🚀

**Comment by amaan-ahmad** [comment]
While fixing this, I found `doMax` and `doMin` were also not exported. So I fixed them in this PR #27 itself.
